Languages/IronPython/IronPython/Modules/_ast.cs

```diff
@@ -876,6 +887,11 @@ public abstract class @operator : AST
         [PythonType]
         public abstract class slice : AST
         {
+            public AstExpression AttachLoc(AstExpression other)
```

Comment:
actually yeah, would you like me to change it in my branch?
and should i squash the whole PR to a single commit?